### DISCUSSION TITLE
Use Music Assistant and HA cast simultaneously

### DISCUSSION THREAD

**Discussion by WebfinityDesign** [music-assistant/support#1244] [Q&A] [answered]
My goal:
Have my morning routine play a radiostation on the TV (with chromecast) while I cast a screen with the weather forecast and calendar appointments for the day using HA cast.

I tried to do this with an automation that starts with Music Assistant followed by calling the HA cast service.
When I do it this way the Music Assistant stops the moment the HA cast starts.
I also tried it the other way around but then I’ve got the same problem the other way around.

Is there a way to let them run simultaneously or is it just not possible?

**Comment by OzGav** [comment]
If you are trying to cast to the same device then no that won’t work as they are both using the cast protocol

**Reply by WebfinityDesign** [reply]
I am indeed trying to cast to the same device. Is there a way around it?
Or would this only be possible when this functionality would be added to Music Assistant?
In that case I'll make a feature request.

**Reply by OzGav** [reply]
I don’t believe any cast device can accept a separate video and audio stream. You are certainly welcome to make a feature request but MA is a music player, there isn’t any plan to do video beyond supplying metadata that the device can use to construct a basic now playing display.

**Reply by WebfinityDesign** [reply]
Ok, thank you for your answer. I'll make a feature request and see what is possible

**Reply by marcelveldt** [reply, marked as answer]
This is simply not possible technically. You can either cast audio/video OR cast a webpage, not both at the same time.
So creating a feature request is a bit a waste of time because it will not work now and will not work ever. I'd suggest you to separate the audio from the video so for instance have a dedicated speaker to play the radio/music

**Comment by tbrasser** [comment]
You could use browser_mod maybe and register it as a cast browser. Not sure if you can use that media player with music assistant in the future.
